core/lib/dal/doc/ProofGenerationDal.md

```diff
@@ -11,12 +11,9 @@ proof_generation_details
 title: Status Diagram
 ---
 stateDiagram-v2
-[*] --> ready_to_be_proven : insert_proof_generation_details
-ready_to_be_proven --> picked_by_prover : get_next_block_to_be_proven
-picked_by_prover --> generated : save_proof_artifacts_metadata
-generated --> [*]
-
-[*] --> skipped : mark_proof_generation_job_as_skipped
-skipped --> [*]
+[*] --> ReadyToBeProven : insert_tee_proof_generation_job
```

Comment:
This is a document for the `proof_generation_dal`, not for the `tee_proof_generation_dal`, so I don't think it should be changed.

Reply:
Ouch, I mixed up the files, sorry!